src/client.ts

```diff
@@ -3305,4 +3306,22 @@ export class StreamChat<
   async getTask(id: string) {
     return this.get<APIResponse & TaskStatus>(`${this.baseURL}/tasks/${id}`);
   }
+
+  /**
+   * deleteChannels - Deletes a list of channel
+   *
+   * @param {string[]} cids Channel CIDs
+   * @param {hard_delete?: boolean} hard_delete Defines if the channel is hard deleted or not
+   *
+   * @return {TaskResponse} A task ID
+   */
+  async deleteChannels(cids: string[], hard_delete?: boolean) {
```

Comment:
to avoid breaking changes in future better to go with 
```suggestion
  async deleteChannels(cids: string[], options?: { hard_delete?: boolean }) {
```

Reply:
This is a good call. Why not we adapt now?